### PR TITLE
ClusteredAnchorGenerator fix, ConcatenatedDataset fix, normalized clustered boxes

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run tests

File: tools/train.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def parse_args():
         action='store_true',
         help='whether to set deterministic options for CUDNN backend.')
     parser.add_argument(
-        '--options', nargs='+', action=DictAction, help='arguments in dict')
+        '--update_config', nargs='+', action=DictAction, help='arguments in dict')
```
Comment (druzhkov-paul) [inline tools/train.py:47]: Why don't you use `options` everywhere instead?

**Review by druzhkov-paul — CHANGES_REQUESTED** [review]

File: tools/train.py (hunk 1 above)
Comment (Ilya-Krylov) [reply tools/train.py:47]: `options` param is differently used in test.py and train.py. That's why I decided to introduce common parameter name for overriding configuration file fields. I can use whatever name but I would like to have the same meaning for train.py and test.py. What do you think?

Comment (druzhkov-paul) [reply tools/train.py:47]: I see. Ok then.

**Review by druzhkov-paul — APPROVED** [review]

**Pull request merged**